clients/python/llmengine/model.py

```diff
@@ -366,3 +368,50 @@ def delete(cls, model: str) -> DeleteLLMEndpointResponse:
         """
         response = cls._delete(f"v1/llm/model-endpoints/{model}", timeout=DEFAULT_TIMEOUT)
         return DeleteLLMEndpointResponse.parse_obj(response)
+
+
+    @classmethod 
+    def download(cls, 
+                 model_name: str, 
+                 download_format: str = "huggingface", 
+                 ) -> ModelDownloadResponse:
+        """
+        Download a fine-tuned model.
+
+        This API can be used to download the resulting model from a fine-tuning job.
+        It takes the `model_name` and `download_format` as parameter and returns a 
+        response object which contains a list of urls associated with the fine-tuned model.
+        The user can then download these urls to obtain the fine-tuned model. If called 
+        on a nonexistent model, an error will be thrown.
+
+        Args:
+            model_name (`str`):
+                name of the fine-tuned model 
+            download_format (`str`): 
+                download format requested (default=huggingface) 
+        Returns:
+            DownloadModelResponse: an object that contains a dictionary of filenames, urls from which to download the model weights
```

Comment:
can you mention those urls are presigned s3 urls and what's the expiration?

Reply:
yes I'll make that explicit!